R/sfi_plot_laqueur_2.R

```diff
@@ -15,163 +15,192 @@ sfi_plot_laqueur_2 <- function(){
   
   # version 1 
-  g1 <- ggplot(data, 
-               aes(x = year, 
-                   y = percent_of_conducted_hearings_resulting_in_a_grant)) +
-    geom_line(size = 1, 
-              alpha = 0.6,
-              linetype = 2) +
+  g1 <- 
+    ggplot(data, 
+           aes(x = year, 
+               y = percent_of_conducted_hearings_resulting_in_a_grant)) +
+    geom_point(size = 4, 
+               alpha = 0.8,
+               color = 'black',
+               stroke = 2,
+               pch = 21) +
+    geom_line(size = 1,
+              linetype = 'dashed',
+              color = '#4D4D4D') +
+    geom_text(aes(label = paste0(percent_of_conducted_hearings_resulting_in_a_grant, 
+                                 '%')),
+                  size = 4,
+              nudge_y = 2,
+              alpha = 0.9) +
+    theme(legend.position="none") +
+    scale_size_area(name = '',
+                    max_size = 5) +
     labs(x = 'Year',
-         y = 'Rate of parole grant',
+         y = 'Hearings resulting in grant',
          title = 'Version 1') +
-    theme_sfi()
+    theme_sfi(lp = 'none')
   
-  # version 2 
-  g2 <- ggplot(data, 
-               aes(x = year, 
-                   y = percent_of_conducted_hearings_resulting_in_a_grant)) +
-    geom_point(size = 3, 
-               alpha = 0.6) +
-    labs(x = 'Year',
-         y = 'Rate of parole grant',
-         title = 'Version 2') +
-    theme_sfi()
+
   
-  # version 3 
-  g3 <- ggplot(data, 
-               aes(x = year, 
-                   y = percent_of_conducted_hearings_resulting_in_a_grant)) +
-    geom_point(size = 3, 
-               alpha = 0.6) +
-    geom_line(size = 1, 
-              alpha = 0.6) +    
+  # version 2
+  g2 <- 
+    ggplot(data, 
+           aes(x = year, 
+               y = percent_of_conducted_hearings_resulting_in_a_grant)) +
+    geom_smooth(method = 'loess',
+                alpha = 0.7,
+                color = 'black',
+                fill = 'black',
+                linetype = 0) + 
+    geom_point(aes(size = percent_of_conducted_hearings_resulting_in_a_grant), 
+               color = 'black',
+               fill = 'white',
+               alpha = 0.8,
+               pch = 21) +
+    geom_text(aes(label = paste0(percent_of_conducted_hearings_resulting_in_a_grant, 
+                                 '%'),
+                  size = percent_of_conducted_hearings_resulting_in_a_grant/8),
+              nudge_x = 0) +
+    scale_size_area(name = '',
+                    max_size = 30) +
+    xlim(c(2007, 2015)) + ylim(c(0, 50)) +
+    theme(legend.position="none") +
     labs(x = 'Year',
-         y = 'Rate of parole grant',
-         title = 'Version 3') +
-    theme_sfi()
+         y = 'Hearings resulting in grant',
+         title = 'Version 2',
+         caption = paste0('Smoothed with a local regression', '\n', 
+                          'with bands representing standard errors')) +
+    theme_sfi(lp = 'none')
+
+  # version 3
+  g3 <- 
```

Comment:
Did they explicitly like the value being shown as both height (y-axis) AND size? If so, good. If not, I say let's lose the `size` argument from aes, since it's duplicated (ie, the value is already shown based on bubble height, no need to use size also).

Reply:
Yes, they did